### PR TITLE
Refactor Nextflow tests

### PR DESCRIPTION
**Pull request by fellen31**
### Added

-

### Changed

- Refactored Nextflow tests

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: tests/cli/workflow/nf_analysis/test_cli.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+"""Tests CLI common methods that Nextflow workflows print help and exits without error."""
```
Comment (henrikstranneheim) [inline tests/cli/workflow/nf_analysis/test_cli.py:1]:
```suggestion
"""Tests CLI common methods of Nextflow workflows."""
```

**Review by henrikstranneheim — APPROVED** [review]
👍 Nice!

**Review by diitaz93 — APPROVED** [review]
💯

File: tests/cli/workflow/nf_analysis/test_cli.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+from cg.cli.workflow.base import workflow as workflow_cli
+from cg.constants import EXIT_SUCCESS, Workflow
+from cg.models.cg_config import CGConfig
+
+
+@pytest.mark.parametrize(
+    "workflow",
+    [Workflow.RAREDISEASE, Workflow.RNAFUSION, Workflow.TAXPROFILER],
+)
+def test_workflow_no_args(cli_runner: CliRunner, workflow: Workflow, request: FixtureRequest):
```
Comment (diitaz93) [inline tests/cli/workflow/nf_analysis/test_cli.py:16]: Possible duplicate PR? https://github.com/Clinical-Genomics/cg/pull/3034/files#diff-38fa23be048298e73afac7dbf12446dfcee2584825ea3620639276a37668b6f2

**Comment by fellen31** [comment]
Closed due to already been merged in PR #3034.

**Pull request closed without merging**